src/main/java/cc/polyfrost/oneconfig/hud/Hud.java

```diff
@@ -208,4 +221,9 @@ public void setScale(float scale, boolean example) {
             name = "Show in GUIs"
     )
     public boolean showInGuis = true;
+
+    @Switch(
+            name = "Show in SkyBlock"
+    )
+    public boolean showInSkyBlock = true;
```

Comment:
Wouldn't there be a way to make this better? Theres no use in this toggle in many scenarios, couldn't we give the developers the option to enable it through a constructor param? 
For existing mods, a hacky fix would be to show the switch for every HUD added by a "Skyblock" OneConfig mod.

Reply:
For some reason people wanted a skyblock toggle for many non-skyblock specific mods like keystrokes